docs/book/src/reference/versions.md

```diff
@@ -85,7 +87,7 @@ These diagrams show the relationships between components in a Cluster API releas
 | Kubernetes v1.26  | ✓                    | ✓                 | ✓                 | ✓                 |
 | Kubernetes v1.27  |                      | ✓                 | ✓                 | ✓                 |
 | Kubernetes v1.28  |                      |                   | ✓                 | ✓                 |
-
+| Kubernetes v1.29  |                      |                   |                   | ✓ >= v1.6.1       |
```

Comment:
q: is there a reason why are not cherry picking similar changes for 1.27 (>= v1.4.2) and 1.28 (>= v1.5.1)?

Reply:
No

Reply:
Should be fixed